packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,4 +1,8 @@
 <template>
+  <template if:true={hasInitializationError}>
```

Comment:
following my question in the js file, should we not have an `if:false` with the `if:true={shouldDisplay}` inside it?